CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.4.3
+- Add authenticator for passwords required as ClearText. Fixes connections issues with Azure Database for PostgreSQL.
+- Added a new boolean parameter in connection as 'allowClearTextPassword' to activate / deactivate the feature.
```

Comment:
Please use only a single description, and also include reference to the PR like the other ones below.